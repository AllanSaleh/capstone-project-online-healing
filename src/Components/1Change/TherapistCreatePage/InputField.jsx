import React from 'react';

const InputField = ({ id, title, inputType, placeholder }) => (
  <div className="my-4">
    <label htmlFor={id} className="text-subtitle capitalize text-black text-opacity-50">
      {title}
    </label>
    <br />
    <input
      className="max-h-8 py-6 px-4 mt-2 w-full border border-black border-opacity-20 rounded-lg outline-none focus:border-blue-dark transition duration-200"
      placeholder={placeholder}
      type={inputType}
      id={id}
      name={id}
      required
    />
  </div>
);

export default InputField;
