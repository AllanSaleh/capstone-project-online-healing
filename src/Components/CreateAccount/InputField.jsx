import React from 'react';

const InputField = ({ id, title, inputType, placeholder }) => (
  <div className="my-4">
    <label
      htmlFor={id}
      className="text-md md:text-subtitle capitalize text-black text-opacity-50"
    >
      {title}
    </label>
    <br />
    <input
      className="text-sm md:text-md lg:text-xl max-h-8 py-3 px-2 md:py-6 md:px-4 mt-2 w-full border border-black border-opacity-20 rounded-lg outline-none focus:border-blue-dark transition duration-200"
      placeholder={placeholder}
      type={inputType}
      id={id}
      name={id}
      required
    />
  </div>
);

export default InputField;
