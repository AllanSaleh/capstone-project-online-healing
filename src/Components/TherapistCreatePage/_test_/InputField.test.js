import * as React from 'react';
import InputField from "../InputField";
import * as renderer from "react-test-renderer";

it('Input Field renders correctly', () => {
  const tree = renderer
    .create(<InputField/>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});