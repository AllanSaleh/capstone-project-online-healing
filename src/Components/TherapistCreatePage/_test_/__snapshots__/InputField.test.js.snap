// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="my-4"
>
  <label
    className="text-md md:text-subtitle capitalize text-black text-opacity-50"
  />
  <br />
  <input
    className="text-sm md:text-md lg:text-xl max-h-8 py-3 px-2 md:py-6 md:px-4 mt-2 w-full border border-black border-opacity-20 rounded-lg outline-none focus:border-blue-dark transition duration-200"
    required={true}
  />
</div>
`;
